src/bindgen/ctyperesolver.rs

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::collections::HashSet;
+
+pub struct CTypeResolver {
+        structs: HashSet<String>,
```

Comment:
Can you decrease the indent here to match the rest of the file?

Reply:
Done!